DependencyInjection/Compiler/AddMapperInformationCompilerPass.php

```diff
@@ -64,5 +64,11 @@ public function process(ContainerBuilder $container)
                 $mapper->addMethodCall('addUnique', array($class, $field, $options));
             }
         }
+
+        foreach (DoctrineCollector::getInstance()->getOverrides() as $class => $overrides) {
```

Comment:
There is a service for that, btw : `data_collector.doctrine` (it's private but IIRC private does not prevent from injecting, just from using it with `Container::get()`